File: Python/POO2/vehiculo/Coche.py
from vehiculo.Vehiculo import Vehiculo


class Coche(Vehiculo):

    def __init__(self, motor):
        super().__init__()
        self.__motor = motor

    def getMotor(self):
        return self.__motor

    def setMotor(self, motor):
        self.__motor = motor

    def quemarRueda(self):
        print("\nEstoy quemando rueda\n")

File: Java/POO2/vehiculo/Coche.java
package vehiculo;

/**
 * Clase Coche hereda de Vehiculo y su atributo y metodo propio
 * @author Juan Manuel Gonzalez Profumo
 * @version 1.0
 */
public class Coche extends Vehiculo {
	private int motor;

	public Coche(int motor) {
		super();
		this.motor = motor;
	}

	public int getMotor() {
		return motor;
	}

	public void setMotor(int motor) {
		this.motor = motor;
	}
	
	void quemarRueda() {
		System.out.println("\nEstoy quemando rueda\n");
	}
}
